test/validation/api/ipsec/ipsec_test_out.c

```diff
@@ -410,8 +411,10 @@ static void test_esp_out_in(struct cipher_param *cipher,
 	printf("\n    %s (keylen %d) %s (keylen %d) ",
 	       cipher->name, cipher_keylen, auth->name, auth_keylen);
 
-	test_out_in_common(false /* ESP */,
-			   cipher->algo, cipher->key,
+	memset(&flags, 0, sizeof(ipsec_test_flags));
```

Comment:
I wonder why you changed the initialization to this memset? I think the previous version (ipsec_test_flags flags = {0}) is better. If the memset were to stay, I would prefer it like this: memset(&flags, 0, sizeof(flags)). But I do not see why not to use the initializer like in the previous version.

Reply:
Couple of stages in CI were complaining. Personally I also prefer it over memset, but I've seen similar issues reported with older toolchain versions. Earlier ODP CI wasn't complaining. But now it is. So I don't think we have an option.

>  I would prefer it like this: memset(&flags, 0, sizeof(flags))
I was just sticking to what was being followed in the rest of the file. Will have it changed.

Reply:
I see.